lib/driver.js

```diff
@@ -172,11 +171,11 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.opts.browserName) {
       log.info('Safari test requested');
-      this.safari = true;      
+      this.safari = true;
       this.opts.app = undefined;
       this.opts.processArguments = this.opts.processArguments || {};
       if (this.isRealDevice()) {
-        this.opts.bundleId = SAFARI_LAUNCHER_BUNDLE;
+        this.opts.bundleId = SAFARI_BUNDLE_ID;
         this._currentUrl = this.opts.safariInitialUrl || `http://appium.io`;
         this.opts.processArguments.args = [this._currentUrl];
```

Comment:
In the past, I did try WebDriverAgent standalone with Simulator & Real device. I can invoke the Safari browser and test with it by this command (don't need `-u `)

```
curl -X POST $JSON_HEADER \
-d "{\"desiredCapabilities\":{\"bundleId\":\"com.apple.mobilesafari\"}}" \
$DEVICE_URL/session
```

So, tomorrow, I'll run appium with iOS 10 simulator to verify